apache/zeppelin
# [WIP] Restful API Editor for interpreters

### What is this PR for?

Add the editor that can be easily access to RESTful API of Interpreter.
The PR is currently being tested and implemented.
### What type of PR is it?

Feature
### Todos
- [x] Implement Restful Api Editor
- [ ] make a new gateway for restful api editor.
- [ ] add license informations for new module.
### What is the Jira issue?

none.
### How should this be tested?

UI will only operate active.
Restful Editor, click the button on left paragraph controller.
Please, See the screen shot.
### Screenshots (if appropriate)

![restfulkylineditor](https://cloud.githubusercontent.com/assets/10525473/19747838/078a01fc-9c19-11e6-8cdd-2b23ed2f3482.gif)
### Questions:
- Does the licenses files need update?
- Is there breaking changes for older versions?
- Does this needs documentation?
